FirebaseCore/Tests/Unit/Swift/FirebaseAppTests.swift

```diff
@@ -15,49 +15,32 @@
 import XCTest
 @testable import FirebaseCore
 
-struct TestApp {
-  static let androidClientID = "correct_android_client_id"
-  static let apiKey = "correct_api_key"
-  static let customizedAPIKey = "customized_api_key"
-  static let clientID = "correct_client_id"
-  static let trackingID = "correct_tracking_id"
-  static let gcmSenderID = "correct_gcm_sender_id"
-  static let googleAppID = "1:123:ios:123abc"
-  static let databaseURL = "https://abc-xyz-123.firebaseio.com"
-  static let storageBucket = "project-id-123.storage.firebase.com"
-  static let deepLinkURLScheme = "comgoogledeeplinkurl"
-  static let newDeepLinkURLScheme = "newdeeplinkurlfortest"
-  static let bundleID = "com.google.FirebaseSDKTests"
-  static let projectID = "abc-xyz-123"
+private extension FirebaseTestConstants.App {
+  static let testAppName1: String = "test_app_name_1"
+  static let testAppName2: String = "test_app_name_2"
 }
 
-let testAppName1 = "test_app_name_1"
-let testAppName2 = "test_app_name_2"
-
-let firebaseDefaultAppName = "__FIRAPP_DEFAULT"
-let googleAppIDKey = "FIRGoogleAppIDKey"
-let firebaseAppNameKey = "FIRAppNameKey"
-let firebaseAppIsDefaultAppKey = "FIRAppIsDefaultAppKey"
-
 class FirebaseAppTests: XCTestCase {
+  let testApp = FirebaseTestConstants.App.self
+  let testOptions = FirebaseTestConstants.Options.self
```

Comment:
@maksymmalyhin After adding the public enum namespace, I tried to make it less verbose to reference a constant. With these two lines, it reduces referencing a constant from `FirebaseTestConstants.Options.clientId` to just `testOptions.clientId` ...thoughts?

Reply:
I personally lean toward using local private declarations of the constants until it is actually required to be shared. So e.g. if the original `TestApp` was moved under `FirebaseAppTests` we would not need to change its usage in the tests. 

If there are clear plans to reuse the constants, then I like the approach 👍